nxcompshad/Makefile.in

```diff
@@ -1,251 +0,0 @@
-#/* All rights reserved.                                                   */
-#/*                                                                        */
-#/* NOTE: This software has received contributions from various other      */
-#/* contributors, only the core maintainers and supporters are listed as   */
-#/* copyright holders. Please contact us, if you feel you should be listed */
-#/* as copyright holder, as well.                                          */
-#/*                                                                        */
-#/**************************************************************************/
-
-#
-# Get values from configure script.
-#
-
-VERSION=@VERSION@
-LIBVERSION=@LIBVERSION@
-
-#
-# We would really like to enable all warnings, -Wredundant-decls,
-# though, gives a warning caused by pthread.h and unistd.h and
-# GCC 3.4 was changed in a way that it now complains about some
-# of the -W directives we used before (-Wmissing-declarations,
-# -Wnested-externs, -Wstrict-prototypes and -Wmissing-prototypes).
-#
-
-CXX         = @CXX@
-CXXFLAGS    = @CXXFLAGS@ @X_CFLAGS@ @DEFS@ \
-              -Wall -Wpointer-arith
-CXXINCLUDES =
-CXXDEFINES  =
-
-CPPFLAGS    = @CPPFLAGS@
-
-#
-# C programs don't share the C++ flags. They should
-# have their own @CCFLAGS@.
-#
-
-CC          = @CC@
-CCFLAGS     = @X_CFLAGS@ @DEFS@ \
-              -Wall -Wpointer-arith
-CCINCLUDES  =
-CCDEFINES   =
-
-LDFLAGS     = @LDFLAGS@
-LIBS        = @LIBS@ -lNX_X11
-
-#
-# Only if THREADS is defined.
-#
-# LIBS = $(LIBS) -lpthread
-#
-
-#
-# Only if you want ElectricFence.
-#
-# LIBS = $(LIBS) -lefence
-#
-
-#
-# Only if you want mpatrol.
-#
-# LIBS = $(LIBS) -lmpatrol -lbfd -liberty
-#
-
-srcdir       = @srcdir@
-prefix       = @prefix@
-datarootdir  = @datarootdir@
-datadir      = @datadir@
-exec_prefix  = @exec_prefix@
-bindir       = @bindir@
-man1dir      = @mandir@/man1
-VPATH        = @srcdir@
-libdir       = @libdir@
-includedir   = @includedir@
-pkgconfigdir = @pkgconfigdir@
-
-INSTALL         = @INSTALL@
-INSTALL_DIR     = $(INSTALL) -d -m 0755
-INSTALL_PROGRAM = @INSTALL_PROGRAM@
-INSTALL_DATA    = @INSTALL_DATA@
-INSTALL_LINK    = cp -av
-DESTDIR         =
-RM_FILE         = rm -f
-RM_DIR          = rmdir -p --ignore-fail-on-non-empty
-
-#
-# This should be autodetected.
-#
-
-MAKEDEPEND     = @MAKEDEPEND@
-DEPENDINCLUDES = -I/usr/include/g++ -I/usr/include/g++-3
-
-.SUFFIXES: .cpp.c
-
-.cpp.o:
-	$(CXX) -c $(CPPFLAGS) $(CXXFLAGS) $(CXXINCLUDES) $(CXXDEFINES) $<
-.c.o:
-	$(CC) -c $(CPPFLAGS) $(CCFLAGS) $(CCINCLUDES) $(CCDEFINES) $<
-
-LIBRARY = Xcompshad
-
-LIBNAME    = lib$(LIBRARY)
-ifeq ($(shell uname),Darwin)
-LIBFULL    = lib$(LIBRARY).$(VERSION).dylib
-LIBLOAD    = lib$(LIBRARY).$(LIBVERSION).dylib
-LIBSHARED  = lib$(LIBRARY).dylib
-COMP_VER   = $(shell echo '$(VERSION)' | cut -d '.' -f 1-3)
-LIBFLAGS   = -install_name $(libdir)/$(LIBLOAD) -compatibility_version $(LIBVERSION) -current_version $(COMP_VER)
```

Comment:
No Darwin builds of nxcompshad should occur. Linux-only at the moment. The Darwin passages in Makefile.in stem from copy+paste from nxcomp IMHO.